tools/scripts/generate.sh

```diff
@@ -22,7 +22,7 @@ echo "# Running transformation based on $OPENAPI_FILE_NAME to the $transformed_f
 cp "$OPENAPI_FOLDER/$OPENAPI_FILE_NAME" "$openapiFileLocation"
 
 npm install
-npm run sdk:transform -- "$openapiFileLocation"
+node ./transformer/src/cli.js transform --input "$openapiFileLocation" --output "$openapiFileLocation"
```

Comment:
FYI: once CLI/Exec section is added we can run cli using package.json instead

Reply:
Sorry not sure if I follow the suggestion

Reply:
You are running cli as node.js application here instead of running it as cli:

https://nodejs.org/en/learn/command-line/run-nodejs-scripts-from-the-command-line

CLI name is usually setup in the package.json.